Brokerages/Exante/ExanteBrokerage.DataQueueHandler.cs

```diff
@@ -0,0 +1,256 @@
+                        }
+
+                        var feedTradesStream = _client.StreamClient.GetFeedTradesStreamAsync(
+                            new[] {ticker},
+                            feedTrade =>
+                            {
+                                var tick = CreateTick(feedTrade);
+                                if (tick != null)
+                                {
+                                    Aggregator.Update(tick);
+                                }
+                            }).SynchronouslyAwaitTaskResult();
+                        if (!feedTradesStream.Success)
+                        {
+                            Log.Error(
+                                $"Exante.StreamClient.GetFeedTradesStreamAsync({ticker}): " +
+                                $"Error: {feedTradesStream.Error}"
+                            );
+                        }
+
+                        _subscribedTickersStreamSubscriptions[ticker] = (feedQuoteStream.Data, feedTradesStream.Data);
+                    }
+                }
+            }
+
+            return true;
+        }
+
+        /// <summary>
+        /// Create a tick from the Exante feed stream data
+        /// </summary>
+        /// <param name="exanteFeedTrade">Exante feed stream data object</param>
+        /// <returns>LEAN Tick object</returns>
+        private Tick CreateTick(ExanteFeedTrade exanteFeedTrade)
+        {
+            var symbolId = exanteFeedTrade.SymbolId;
+            if (!_subscribedTickers.TryGetValue(symbolId, out var symbol))
+            {
+                // Not subscribed to this symbol.
+                return null;
+            }
+
+            if (exanteFeedTrade.Size == decimal.Zero)
+            {
+                return null;
+            }
+
+            var utc = exanteFeedTrade.Date;
+            // Convert the timestamp to exchange timezone and pass into algorithm
+            var time = utc.ConvertTo(DateTimeZone.Utc, TimeZones.NewYork);
+
+            var instrument = _client.GetSymbol(symbolId);
+
+            var size = exanteFeedTrade.Size ?? 0m;
+            var price = exanteFeedTrade.Price ?? 0m;
+            return new Tick(time, symbol, "", instrument.Data.Exchange, size, price);
+        }
+
+        /// <summary>
+        /// Create a tick from the Exante tick shorts stream data
+        /// </summary>
+        /// <param name="exanteTickShort">Exante tick short stream data object</param>
+        /// <returns>LEAN Tick object</returns>
+        private Tick CreateTick(ExanteTickShort exanteTickShort)
+        {
+            if (!_subscribedTickers.TryGetValue(exanteTickShort.SymbolId, out var symbol))
+            {
+                // Not subscribed to this symbol.
+                return null;
+            }
+
+            var bids = exanteTickShort.Bid.ToList();
+            var asks = exanteTickShort.Ask.ToList();
+            return new Tick(exanteTickShort.Date, symbol, "", "",
```

Comment:
Should follow the same patter for `exanteTickShort.Date` as for Trade data, similar comments here as above 👍

Reply:
ping